tests/ui/asm/aarch64/type-check-3.rs

```diff
@@ -1,5 +1,5 @@
 // only-aarch64
```

Comment:
These haven't gotten converted

Reply:
I hadn't yet implemented the `only` and `ignore` things, looking into that now.